src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +22,14 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
```

Comment:
```suggestion
        return numbers.stream()
```
це питання стилю, не є помилкою. Особисто я більше зустрічала коду коли метод stream()викликають на попередньому рядку, а інші уже з нового